lib/node_modules/@stdlib/math/base/special/negalucas/src/main.c

```diff
@@ -0,0 +1,124 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/negalucas.h"
+#include "stdlib/math/base/special/abs.h"
+#include "stdlib/constants/float64/max_safe_nth_lucas.h"
+
+static const int32_t STDLIB_CONSTANT_FLOAT64_MAX_SAFE_NTH_LUCAS;
```

Comment:
As you are including "stdlib/constants/float64/max_safe_nth_lucas.h" you do not need to explicitly define `STDLIB_CONSTANT_FLOAT64_MAX_SAFE_NTH_LUCAS`
```suggestion
```